net-irc/polari/polari-46.0.ebuild

```diff
@@ -0,0 +1,56 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit gnome.org gnome2-utils meson xdg
+
+DESCRIPTION="An IRC client for GNOME"
+HOMEPAGE="https://wiki.gnome.org/Apps/Polari https://gitlab.gnome.org/GNOME/polari"
+
+LICENSE="GPL-2+ CC0-1.0 CC-BY-SA-4.0"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~x86"
+IUSE="test"
+RESTRICT="!test? ( test )"
+
+# GTK3 still used by thumbnailer: https://gitlab.gnome.org/GNOME/polari/-/issues/223
+DEPEND="
+	x11-libs/gtk+:3[introspection]
+	gui-libs/gtk:4[introspection]
+	gui-libs/libadwaita:1[introspection]
+	>=dev-libs/glib-2.43.4:2
+	net-libs/telepathy-glib[introspection]
+	>=dev-libs/gobject-introspection-1.50:=
+	>=dev-libs/gjs-1.73.1
+
+	x11-libs/gdk-pixbuf:2[introspection]
+	>=app-text/gspell-1.4.0[introspection]
+	x11-libs/pango[introspection]
+	app-crypt/libsecret[introspection]
+	net-libs/libsoup:3.0[introspection]
+	net-im/telepathy-logger[introspection]
```

Comment:
`telepathy-logger` was removed in `46.0`:
  - https://gitlab.gnome.org/GNOME/polari/-/blob/46.0/NEWS?ref_type=tags#L4
  - https://gitlab.gnome.org/GNOME/polari/-/merge_requests/264

Reply:
Done in https://github.com/gentoo/gentoo/commit/7d6cff0bdf173dbed8631c88dcebe0e5cb0236b3